### PR TITLE
add node selector to admission-controllers

### PR DESCRIPTION
**Pull request by xilabao**
see https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/podnodeselector/admission.go#L40
https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/podnodeselector/admission.go#L67

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3691)

### CODE REVIEW

File: docs/admin/admission-controllers.md [hunk 1]
```diff
@@ -257,6 +257,33 @@ storage classes and how to mark a storage class as default.
 This plug-in sets the default forgiveness toleration for pods, which have no forgiveness tolerations, to tolerate
 the taints `notready:NoExecute` and `unreachable:NoExecute` for 5 minutes.
 
+### PodNodeSelector
+
+This plug-in defaults and limits what node selectors may be used within a namespace by reading a namespace annotation and a global configuration.
+
+#### Configuration File Format
```
Comment (derekwaynecarr) [inline docs/admin/admission-controllers.md]: can you add some text that notes that the configuration file format will move to a versioned file in a future release?

Comment (xilabao) [inline docs/admin/admission-controllers.md]: Done.

**Comment by xilabao** [comment]
ping @derekwaynecarr @aveshagarwal @chenopis

**Comment by aveshagarwal** [comment]
@xilabao I will review it soon.

**Comment by chenopis** [comment]
@derekwaynecarr Can you give your approval based on the changes you requested? We can still wait for @aveshagarwal to review.

**Comment by chenopis** [comment]
@aveshagarwal Can you take a look when you get a chance? Thx

**Comment by aveshagarwal** [comment]
@chenopis yeah sure, sorry for delay.

**Comment by chenopis** [comment]
@aveshagarwal No worries. Thx!

**Comment by aveshagarwal** [comment]
I reviewed, and changes  LGTM

**Comment by xilabao** [comment]
@chenopis

**Pull request merged**